### PR TITLE
A11Y: switch from links to buttons when pertinent

### PR DESCRIPTION
**Pull request by davidbgk**
CSS part is missing to be able to actually toggle to the `button` element. But some choices had to be discussed/made prior to that.

### CODE REVIEW

File: umap/static/umap/js/umap.core.js [hunk 1]
```diff
@@ -286,14 +286,25 @@ L.DomUtil.createFieldset = (container, legend, options) => {
 }
 
 L.DomUtil.createButton = (className, container, content, callback, context) => {
+  // TODO: actually switch to buttons’ elements.
   const el = L.DomUtil.add('a', className, container, content)
   el.href = '#'
+  // const el = L.DomUtil.add('button', className, container, content)
```
Comment (davidbgk) [inline umap/static/umap/js/umap.core.js:292]: This is the line to uncomment to get the full button experience 😅

**Comment by yohanboniface** [comment]
Our buttons seems grey instead of white, like buttons comings from Leaflet/plugins:

image

I'd say we should keep them white, or make them all grey.

**Comment by yohanboniface** [comment]
This button text seems to long to me:

image

Save/Enregistrer already says enough, no need to add "current changes" imho. :)

**Comment by yohanboniface** [comment]
If we touch the labels of those buttons, a bit of context to take into account: https://github.com/umap-project/umap/issues/556

**Comment by yohanboniface** [comment]
Broken picto:

image

**Review by yohanboniface — APPROVED** [review]
Let's go!

**Pull request merged**